### PR TITLE
blog: Add mTLS everywhere blog post

### PR DESCRIPTION
**Pull request by nfrankel**
Screenshot 2023-03-23 at 10 39 20

### CODE REVIEW

**Comment by nfrankel** [comment]
Please upload the two images to API7 cloud and change the references in the post

**Comment by nfrankel** [comment]
Preview: https://deploy-preview-1539--apache-apisix.netlify.app/blog/2023/03/23/mtls-everywhere/

**Review by moonming — DISMISSED** [review]

File: blog/en/blog/2023/03/23/mtls-everywhere.md [hunk 1]
```diff
@@ -0,0 +1,584 @@
...
+3. Configure _server_ authentication for the Admin API
+4. Regular HTTPS port
+5. Admin HTTPS port
+6. Certificates for server authentication
+7. Certificates for client authentication
+8. Two sets of certificates are used, one for server authentication for the Admin API and regular HTTPS, and one for client authentication for etcd.
+
+At this point, we can apply the above manifests and see the two pods communicating. When connecting, Apache APISIX sends its `apisix-client` certificate via HTTPS. Because an authority signs the certificate that etcd trusts, it allows the connection.
+
+I've omitted the `Service` definition for brevity's sake, but you can check them in the associated [](https://github.com/ajavageek/tls-apisix).
```
Comment (Neilblaze) [inline blog/en/blog/2023/03/23/mtls-everywhere.md]: Suggested change:

```md
I've omitted the `Service` definition for brevity's sake, but you can check them in the [associated](https://github.com/ajavageek/tls-apisix).
```

**Review by Neilblaze — APPROVED** [review]
LGTM! 🎉

**Comment by nfrankel** [comment]
@Neilblaze thanks for your review!

**Review by SkyeYoung — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**